### DISCUSSION TITLE
Any news about the TilesList

### DISCUSSION THREAD

**Discussion by blendsdk** [microsoft/fluentui#17685] [Q&A] [answered]
Hi,

Is there are news or any plans for adding the `react-experiments/TitlesList` to the official packages? In case the `TitlesList` is not a priority yet, then I would very much appreciate a common component "template" for implementing custom components that would fit in the `@fluentui` architecture. Perhaps I can try to implement (or migrate) the `TitlesList` myself.

Thanks.

Please check this [link](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/10546/merge/experiments/dist/index.html#/examples/tileslist)

**Comment by JustSlone** [comment, marked as answer]
Sorry, it's not currently on the roadmap to migrate TitlesList over. 

I agree with what you said on the common component "template" for implementation custom components. We're working on updating the Fluent UI architecture right now, unfortunately the pattern is not stable right now, but our goal once it's stabilized is to encourage more community development of components. 

soon™️ 😄

**Reply by blendsdk** [reply]
Thank you. But isn't there anything you could help me out here? I am sure the structure of one of these components is stable enough to be used as a model for another component.
